.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+
+name: Release
+
+on:
+  pull_request:
+    branches: [master]
+    types: [closed]
+
+jobs:
+  documentation:
+    if: ${{ github.event.pull_request.merged == true }}
+    name: Check Docs & Overrides
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Node.js 16.7
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.7
+
+      - name: Install NPM   
+        run: npm install
+
+      - name: Check Docs
+        run: npm run export:docs
+      - name: Check Overrides
+        run: npm run export:overrides
+
+  release:
+    if: ${{ github.event.pull_request.merged == true }}
```

Comment:
In which case this would change.
Also it would be good to ensure that this only runs on the cloudflare owner.

Reply:
Done.